icons/webassembly/webassembly-original.svg

```diff
@@ -0,0 +1 @@
+<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M78.64 0v.691c0 8.106-6.57 14.676-14.675 14.676-8.106 0-14.676-6.574-14.676-14.676V0H0v128h128V0H78.64z" fill="#654FF0"/><path d="M29.734 68.98h8.48l5.794 30.844h.105l6.961-30.843h7.934l6.285 31.222h.125l6.601-31.222h8.32L69.528 114.3H61.11l-6.234-30.844h-.164l-6.672 30.844h-8.578l-9.727-45.32zm60.168 0h13.375l13.282 45.321h-8.754l-2.887-10.086H89.684l-2.23 10.086H78.93l10.972-45.32zm5.09 11.172-3.699 16.625h11.516l-4.246-16.625h-3.57z" fill="#fff"/></svg>
```

Comment:
Please make the `WA` text transparent.

Reply:
@Panquesito7 in both of the svg or just in the `original` one?

Reply:
Both SVGs, please.

Reply:
@Utkarsh1504 and @Panquesito7 I think just the original version is fine. Since both are the similar, we can actually use the `original` version for the `plain` version as well.

So pretty much:
- Make the WA text in the `original` version transparent
- Delete the `plain` version
- Update the `devicon.json` so it includes the `original` as a `font` friendly version